scripts/codespell.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+# Ignore vendor folder and check file names as well
+# Note: ignore "ba" in https://github.com/k3s-io/k3s/blob/4317a91/scripts/provision/vagrant#L54
+codespell --skip=.git,./vendor --check-filenames --ignore-words-list=ba
+
+code=$?
+if [ $code -ne 0 ]; then
+  echo "Error: codespell found one or moe problems!"
```

Comment:
@jenting is there any way you _meant_ to do this `moe` typo?

Reply:
https://github.com/k3s-io/k3s/commit/57041f02392eb7c7c3c72423b98dbb6717d2da55#r45485230

Reply:
my bad, good catch!

Reply:
I know this PR was merged.  It has 3 reviewers, and apparently they all missed this?!?!